### DISCUSSION TITLE
SumatraPDF not printing the correct page

### DISCUSSION THREAD

**Discussion by zohozer** [sumatrapdfreader/sumatrapdf#4403] [Q&A] [answered]
Hi all. I do try to print a single page from a PDF document that consist in multiple pages and always I do get printed the wrong page. I just tried to print the page nr. 4 from a document containing four pages and instead I do get into the printer the page nr. 1. If I disable the "Let the app change my printing preferences" I got printed the page nr. 2.

I am using Windows 11 and I tested SumatraPDF v3.5.2 and the v3.6pre-release and it is doing the same.

**Comment by GitHubRulesOK** [comment, marked as answer]
Seems to be a Windows 11 problem as only affects users with MS new printing dialog 
there are "issues" already reported for monitor any progress
https://github.com/sumatrapdfreader/sumatrapdf/issues/4235 
and earlier https://github.com/sumatrapdfreader/sumatrapdf/issues/3453 and https://github.com/sumatrapdfreader/sumatrapdf/issues/3427 and earlier https://github.com/sumatrapdfreader/sumatrapdf/issues/3216#issuecomment-1378880159
